Permanent/Modules/EditMetadata/ViewModels/FilesMetadataViewModel.swift

```diff
@@ -9,12 +9,14 @@ import Foundation
 
 protocol GenericViewModelProtocol: ObservableObject {
```

Comment:
There is no need for the this protocol as far as I see.

Reply:
In SectionHeaderView, instead of sending a viewModel, you can just send the selectedFiles:

`SectionHeaderView(selectedFiles: viewModel.selectedFiles)`

```
struct SectionHeaderView: View {
    @State var selectedFiles: [FileModel]?
```

```
if let files = selectedFiles {
                    if !files.isEmpty, let url = URL(string: files.first?.thumbnailURL500) {
```

Reply:
In TagsView if we really need the viewModel to get the completion for delete, just send the FilesMetadataViewModel.